### PR TITLE
Support connections to EnterpriseDB Postgres databases.

### PR DESCRIPTION
**Pull request by al3x**
[EnterpriseDB](http://www.enterprisedb.com/) is a distribution of Postgres. Knex should be compatible with EnterpriseDB as-is, but because it's looking for the string `Postgres` in the result of `SELECT version();`, connections to EnterpriseDB databases currently fail.

### CODE REVIEW

**Comment by elhigu** [comment]
@al3x Looks like tests are failing any ideas why?

**Comment by al3x** [comment]
@elhigu Not sure. Most Postgres tests are passing?

File: src/formatter.js [hunk 1]
```diff
@@ -86,6 +86,10 @@ assign(Formatter.prototype, {
     return this._wrapString(value + '');
   },
 
+  wrapAsIdentifier: function wrapAsIdentifier(value) {
+    return this.client.wrapIdentifier((value || '').trim());
+  },
+
```
Comment (elhigu) [inline src/formatter.js]: Why these changes are here?

**Comment by elhigu** [comment]
Implemented in #1993
